<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_admin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>951</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date of Birth</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Username</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Password</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Delete</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ADMINISTRATORS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_search">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>530</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="number">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>23</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_to_excel"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_us"/>
   </widget>
   <widget class="QMenu" name="menuOption_2">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionSign_out_2"/>
    <addaction name="actionExit_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
   <addaction name="menuOption_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSign_out">
   <property name="text">
    <string>Sign out</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExport_to_excel">
   <property name="text">
    <string>Export to excel</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About us</string>
   </property>
  </action>
  <action name="actionSign_out_2">
   <property name="text">
    <string>Sign out</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
